cf_xarray/accessor.py

```diff
@@ -1635,6 +1675,10 @@ def get_associated_variable_names(
                 if dbounds:
                     coords["bounds"].append(dbounds)
 
+        if not skip_grid_mappings:
+            if "grid_mapping" in attrs_or_encoding:
+                coords["grid_mapping"] = [attrs_or_encoding["grid_mapping"]]
+
```

Comment:
I don't know that we need `skip_grid_mappings` unless you have a use case.

`bounds` are funny because sometimes bounds have the same `standard_name` and in some cases, we know the user doesn't want that.